### PR TITLE
OD-13 [Feature] Integration the new default variable into SMS, Email verefication

### PR DESCRIPTION
**Pull request by AndrRyaz**
@sofiiakvasnevska 
## Issue
OD-13 https://weboo.atlassian.net/jira/software/projects/OD/boards/9?selectedIssue=OD-13
## Description
Email, SMS verification.
Pass new variable for the default name into DS Provider
## Screenshots/screencasts
https://screencast-o-matic.com/watch/cY63jM7Pbq
## Backward compatibility
This change is fully backward compatible.
## Reviewers
@upplabs-alex-levchenko @YaroslavOvdii

### CODE REVIEW

File: js/interface.js [hunk 1]
```diff
@@ -103,6 +103,7 @@ switch (data.type) {
     dsQueryData = {
       settings: {
         dataSourceLabel: 'Select the data source containing the user information',
+        defaultDataSourceName: 'SMS verification data for ' + Fliplet.Env.get('appName'),
```
Comment (tonytlwu) [inline js/interface.js]:
```suggestion
        default: {
          name: 'User data for ' + Fliplet.Env.get('appName')
        },
```

File: js/interface.js [hunk 2]
```diff
@@ -124,6 +125,7 @@ switch (data.type) {
     dsQueryData = {
       settings: {
         dataSourceLabel: 'Select the data source containing the user information',
+        defaultDataSourceName: 'Email verification data for ' + Fliplet.Env.get('appName'),
```
Comment (tonytlwu) [inline js/interface.js]:
```suggestion
        default: {
          name: 'User data for ' + Fliplet.Env.get('appName')
        },
```

**Review by tonytlwu — CHANGES_REQUESTED** [review]
@AndrRyaz See https://github.com/Fliplet/fliplet-widget-data-source-query/pull/20#discussion_r510193868

File: js/interface.js [hunk 3]
```diff
@@ -139,7 +143,7 @@ switch (data.type) {
 }
 
 // Open data source query provider inline
-var dsQueryProvider = Fliplet.Widget.open('com.fliplet.data-source-query', {
+var dsQueryProvider = Fliplet.Widget.open('com.fliplet.data-source-query-andrey', {
```
Comment (tonytlwu) [inline js/interface.js]: @upplabs-alex-levchenko Please explain why we are asked to review the PR when it contains this error?

**Review by tonytlwu — CHANGES_REQUESTED** [review]

File: js/interface.js (hunk 3 above)
Comment (upplabs-alex-levchenko) [reply js/interface.js]: @tonytlwu Sometimes even I make mistakes, didn't you? :)

File: js/interface.js [hunk 4]
```diff
@@ -102,7 +102,10 @@ switch (data.type) {
   case 'sms':
     dsQueryData = {
       settings: {
-        dataSourceLabel: 'Select the data source containing the user information',
+        dataSourceTitle: 'Select the data source containing the user information',
```
Comment (tonytlwu) [inline js/interface.js]:
```suggestion
        dataSourceTitle: 'User list data source',
```

File: js/interface.js [hunk 5]
```diff
@@ -123,7 +126,10 @@ switch (data.type) {
   default:
     dsQueryData = {
       settings: {
-        dataSourceLabel: 'Select the data source containing the user information',
+        dataSourceTitle: 'Select the data source containing the user information',
```
Comment (tonytlwu) [inline js/interface.js]:
```suggestion
        dataSourceTitle: 'User list data source',
```

**Review by tonytlwu — CHANGES_REQUESTED** [review]
Let's shorten and simplify the title

File: js/interface.js (hunk 5 above)
Comment (AndrRyaz) [reply js/interface.js]: Done

**Review by tonytlwu — APPROVED** [review]
@sofiiakvasnevska This is ready for testing

Risk: Low
Affected: Email and SMS verification components
Verify: Data source integration works correctly

Squash and merge: Updates default data source names and title

**Pull request merged**